documentation/src/test/java/org/hibernate/search/documentation/gettingstarted/withhsearch/withoutanalysis/GettingStartedWithoutAnalysisIT.java

```diff
@@ -79,6 +84,24 @@ public void test() {
 			bookIdHolder.set( book.getId() );
 		} );
 
+		OrmUtils.withinJPATransaction( entityManagerFactory, entityManager -> {
+			// tag::manual-index[]
+			// Not shown: get the entity manager and open a transaction
+			FullTextEntityManager fullTextEntityManager = Search.getFullTextEntityManager( entityManager ); // <1>
+
+			MassIndexer indexer = fullTextEntityManager.createIndexer( Book.class ); // <2>
```

Comment:
I think we should at least hint at the fact that there are options that users can fine-tune. Maybe add a call to `threadsToLoadObjects` and a note saying what it does, and mentioning there are other options? You can also redirect to the dedicated section for more details: `<<manual-index-changes>>`